app/decorators/middleware_deployment_decorator.rb

```diff
@@ -1,4 +1,14 @@
 class MiddlewareDeploymentDecorator < MiqDecorator
+  def self.fonticon
+    if name.end_with? '.ear'
```

Comment:
this is problematic .. you can use `name` in the non-self version, but in `self.fonticon` you can't (because there's no instance) .. so I guess just `fa fa-file-text-o` here?